### DISCUSSION TITLE
Development build fail after dependency updates (validatedIncludeSpecs)

### DISCUSSION THREAD

**Discussion by andypimlett** [adonisjs/core#2291] [Help] [answered]
Hitting a build fail following deps update. Terminal killed and reset everything but consistently failing.
Appreciate any hints if anyone has seen this, thanks?

```
[ info ]  building project...
[ info ]  starting http server...
[ error ]  Cannot read property 'validatedIncludeSpecs' of undefined
    at Watcher.initiateSourceFileManager (.../node_modules/@poppinss/chokidar-ts/build/src/Watcher.js:56:49)
```

**Comment by andypimlett** [comment]
Looks like it's something in peer dependencies. Reverted package-lock and all is fine; so something has been de-stabilised somewhere.

**Reply by targos** [reply]
That's introduced by TypeScript 4.2.

**Comment by RomainLanz** [comment, marked as answer]
Hey! 👋 

The latest release of TypeScript broke something in one of our dependency.
Please, downgrade your typescript version to `~4.1` in your `package.json`.

**Reply by LeCoupa** [reply]
So frustrating that Github Discussions are so badly indexed! I was trying to figure out what happened for one hour. Thank you @RomainLanz
